bin.src/makeCoaddTempExp.py

```diff
@@ -20,9 +20,5 @@
 # the GNU General Public License along with this program.  If not,
 # see <http://www.lsstcorp.org/LegalNotices/>.
 #
-import lsst.pex.logging as pexLog
 from lsst.pipe.tasks.makeCoaddTempExp import MakeCoaddTempExpTask
-
-pexLog.Trace.setVerbosity('lsst.coadd', 3)
-pexLog.Trace.setVerbosity('lsst.ip.diffim', 1)
```

Comment:
That's a great question and I'm afraid I don't know the answer. I had assumed they provided an appropriate level of verbosity, but perhaps it is too much.